app/(landing)/blog/[slug]/page.tsx

```diff
@@ -118,14 +120,29 @@ export async function generateMetadata({ params }: BlogDetailProps): Promise<Met
   }
 }
 
+export const getCounterView = async (postId: Number) => {
+	try {
+		const responseIncrement = await client.mutate({
+			mutation: INCREMENT_POST_VIEWS_MUTATION,
+			variables: { postId: postId },
```

Comment:
```suggestion
			variables: { postId: postId },
			context: {
                            fetchOptions: {
                              cache: 'no-store',
                            },
                          },
```

Because you used the default option for cache configuration via next, that's why you couldn't recall the API after every time you F5.

NextJS-14: Default option is force-cache
NextJS-15: Default option is no-cache

Reply:
Ref this link: https://nextjs.org/docs/app/building-your-application/data-fetching/fetching

Reply:
I think it's working now. The ways you guys testing are incorrect.

![image](https://github.com/user-attachments/assets/ae18c221-6889-4990-821a-4e3a854c217b)

Reply:
@hoainho This is unrelated to the Next.js fetch feature since we're using Apollo to handle GraphQL requests.

As mentioned earlier, the page executes a GraphQL query to increment the views, so it isn't connected to Next.js caching strategies.

Reply:
and why do we need to increase the postviews if the user change the page, then back to the previous?